sirepo/template/cloudmc.py

```diff
@@ -577,6 +584,26 @@ def _generate_range(filter):
     return "numpy.{}({}, {}, {})".format(space, start, stop, filter.num)
 
 
+def _generate_run_mode(data, v):
+    r = data.models.openmcAnimation.jobRunMode
+    cores = 1 if r == "sequential" else sirepo.mpi.cfg().cores
+    if v.isPythonSource:
+        cores = 0
+    if data.models.settings.varianceReduction == "weight_windows_tally":
+        if v.isSBATCH:
+            raise AssertionError("Weight Windows are not yet available with sbatch")
```

Comment:
This is fine, but I think there should also be equivalent logic on the client side to disable running with sbatch if this setting is in place.